rest/handler.go

```diff
@@ -79,9 +79,10 @@ type handler struct {
 type handlerPrivs int
 
 const (
-	regularPrivs = iota // Handler requires authentication
-	publicPrivs         // Handler checks auth but doesn't require it
-	adminPrivs          // Handler ignores auth, always runs with root/admin privs
+	regularPrivs              = iota // Handler requires authentication
+	publicPrivs                      // Handler checks auth but doesn't require it
```

Comment:
Let's change the comment to be more clear about the difference between this and the new option

```suggestion
	publicPrivs                      // Handler ensures valid auth if present, but doesn't require it
```